depends/windowsstore/nasmw/CMakeLists.txt

```diff
@@ -1,7 +1,18 @@
 cmake_minimum_required(VERSION 3.5)
 project(nasmw)
 
-install(
-	DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
-	DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
-)
+# Prevent this on UWP ARM build
+if(NOT _ARM_)
```

Comment:
@Rechi is there another way possible to prevent this use on ARM by WinStore?

Berfore was only a already compiled App used where hast prevent this compile problem.